.gitignore

```diff
@@ -3,3 +3,5 @@
 .vscode
 node_modules
 .envrc
+admin-ui/package-lock.json
+mealplanner-ui/src/pages/MealPlans/__generated__/MealPlansQuery.graphql.ts
```

Comment:
Should not git ignore generated file. Please commit this file.

Reply:
Removed files from the .gitignore. Thank you

Reply:
sorry, I believe is fixed now. Thank you.